src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -156,6 +156,13 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewIni
 
   public tabIndex: number = -1;
 
+  /**
+   * Specifies a familiar object for this repeater item to be returned when a `orderChange` event fires.
+   * This property is required if `reorderable` is set to `true`.
```

Comment:
As noted below, I think descriptions should usually be on a single line based on looking at popover, but I think it makes sense to break out conditional information like this. That said, I think we should either emphasize the conditional information by making this a sentence fragment that starts with "Required if" or rearrange the sentence to put the conditional "if" clause at the start. Personally, I'd go with the first suggestion and change this to "Required if you set the `reorderable` property to `true`." But if that is problematic, we could flip the order to: "If you set the `reorderable` property to `true`, this property is required."

Reply:
> Required if you set the reorderable property to true.
I like that. Changed.